Llama OOMs in CI

Openllama 3B test fails in CI with error
```
tests/jax/models/llama/openllama_3b_v2/test_openllama_3b_v2.py::test_openllama3b_inference 
Error: Process completed with exit code 137.
```
The root cause is that the system can take ~34Gb of RAM at it's peak during the compile, while our CI machines have 32Gb.

--- a/tests/jax/models/llama/openllama_3b_v2/test_openllama_3b_v2.py
+++ b/tests/jax/models/llama/openllama_3b_v2/test_openllama_3b_v2.py
@@ -26,7 +26,8 @@ def training_tester() -> LLamaTester:
 # ----- Tests -----
 
 
-@pytest.mark.xfail(reason="failed to legalize operation 'stablehlo.reduce'")
+# @pytest.mark.xfail(reason="failed to legalize operation 'stablehlo.reduce'")
+@pytest.mark.skip(reason="OOMs in CI")
 def test_openllama3b_inference(
     inference_tester: LLamaTester,
 ):

--- a/tests/jax/models/llama/openllama_3b_v2/test_openllama_3b_v2.py
+++ b/tests/jax/models/llama/openllama_3b_v2/test_openllama_3b_v2.py
@@ -27,7 +27,9 @@ def training_tester() -> LLamaTester:
 
 
 # @pytest.mark.xfail(reason="failed to legalize operation 'stablehlo.reduce'")
-@pytest.mark.skip(reason="OOMs in CI")
+@pytest.mark.skip(
+    reason="OOMs in CI (https://github.com/tenstorrent/tt-xla/issues/186)"
+)
 def test_openllama3b_inference(
     inference_tester: LLamaTester,
 ):

